cli/cli_app.go

```diff
@@ -319,6 +319,11 @@ func RunTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		terragruntOptions.DownloadDir = terragruntConfig.DownloadDir
 	}
 
+	// Override the default value of retryable errors using the value set in the config file
+	if terragruntConfig.RetryableErrors != nil {
+		terragruntOptions.RetryableErrors = terragruntConfig.RetryableErrors
```

Comment:
Not for this PR, but we should probably expose a helper function that returns all the default retryable errors. I think 90% of use cases for this will want to append to the default list, rather than replace. If we had a helper (e.g. `get_default_retryable_errors`), then one could do the append use case with:

```
retryable_errors = concat(get_default_retryable_errors(), ["my special snowflake"])
```

Reply:
Agreed, that makes total sense.

Reply:
@infraredgirl Could you file an issue to track this?

Reply:
Done: https://github.com/gruntwork-io/terragrunt/issues/1383